### DISCUSSION TITLE
Add uptime since last battery charge to the battery setting

### DISCUSSION THREAD

**Discussion by devnoname120** [InfiniTimeOrg/InfiniTime#1613] [Ideas]
It could show:
- Uptime since last time the charger was plugged.
- Screen on time (SOT) since last time the charger was plugged.

It would be useful to know how long the PineTime holds, and it could pave the way to battery life estimations.

**Comment by Zandengoff** [comment]
Was going to put in this feature request and saw that had already been done. Would love this on the main battery screen.
